### PR TITLE
Pithos

### PR DESCRIPTION
**Pull request by obadz**
- move from own file to python-packages.nix
- upgrade from 1.0.1 to 1.1.1

### CODE REVIEW

**Comment by domenkozar** [comment]
If it's an application (not a library), it shouldn't be inside `python-packages.nix`

**Comment by obadz** [comment]
Very well.

**Comment by lowfatcomputing** [comment]
[The icons etc](https://github.com/pithos/pithos/tree/master/data) are missing it would be cool if you could install those too.

**Comment by obadz** [comment]
@lowfatcomputing, the only icon is the system tray one. It shows up fine in my tray.. does it not work for you?

**Comment by obadz** [comment]
@domenkozar, anything else that needs changing before merging?

File: pkgs/applications/audio/pithos/default.nix [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
   meta = with stdenv.lib; {
     description = "Pandora player";
-
-    longDescription = ''
-      Pandora Internet Radio player for GNOME
-    '';
-
-    homepage = http://pithos.github.io/ ;
-
+    longDescription = "Pandora Internet Radio player for GNOME";
```
Comment (domenkozar) [inline pkgs/applications/audio/pithos/default.nix]: this is `description`

**Comment by obadz** [comment]
Fixed

**Comment by lowfatcomputing** [comment]
As for my top-row button-icons issue, it looks like someone will need to package [gnome-icon-theme-symbolic](https://github.com/GNOME/gnome-icon-theme-symbolic) at some point and add it as a dependency to pithos:

https://github.com/pithos/pithos/issues/137#issuecomment-59370450

2015-09-25-100735_847x65_scrot

**Comment by obadz** [comment]
@lowfatcomputing, could you try adding gnome3.adwaita-icon-theme to your systemPackages and see if that fixes it?

**Comment by lowfatcomputing** [comment]
@obadz, That seemed to have worked. Thanks!

**Comment by obadz** [comment]
The GTK/Gnome stuff is kind of confusing. I don't know what the right way to tackle this better is..

**Comment by geerds** [comment]
@lowfatcomputing @obadz Instead of manually call `wrapProgram` you should use the [wrapGAppsHook](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/wrap-gapps-hook.sh). I've already fixed this: 93adb22c6faaaa3100882409daf11877ea672885 (also a few other nitpicks, see current file in master branch)

Thanks for your contribution!

**Comment by geerds** [comment]
@lowfatcomputing Could you please open an issue regarding the missing `gnome-icon-theme-symbolic`?

**Comment by obadz** [comment]
@geerds, thanks.

**Pull request merged**